frontend/lib/framework/error-classes-list.cpp

```diff
@@ -537,4 +537,11 @@ void ErrorProcDefExplicitAnonFormal::write(ErrorWriterBase& wr) const {
   wr.code(fn, {formal});
 }
 
+void ErrorSuperFromTopLevelModule::write(ErrorWriterBase& wr) const {
+  auto use = std::get<const uast::AstNode*>(info);
+  wr.heading(kind_, type_, use, "cannot use/import super from a toplevel "
+                                "module");
```

Comment:
Needs a period (`"module."`). Also "toplevel" => "top level"?. Also also, let's 'quote' `use` and `import`, since they are Chapel syntax terms.